buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/ElasticsearchNode.java

```diff
@@ -107,9 +142,24 @@ public void setDistribution(Distribution distribution) {
         this.distribution = distribution;
     }
 
+    public void plugin(URI plugin) {
+        requireNonNull(plugin, "Plugin name can't be null");
+        checkFrozen();
+        this.plugins.add(plugin);
+    }
+
+    public void plugin(String plugin) {
```

Comment:
Why do we need so many variations? Can we please have 1 method so there is never a question of which method should be used?

Reply:
These are meant to make this easier to be passed from the DSL. 
I will remove the version with `String`, reading this it has some ambiguity to it as it's generally converted to a `File` in Gradle, but would like to keep the method with `File` 
to make it easier to pass `file(...)` from the DSL like the tests do.